acct-user/monitorix/monitorix-0.ebuild

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="Monitorix system tool user"
+ACCT_USER_ID=401
+ACCT_USER_HOME=/var/lib/monitorix
```

Comment:
Is this needed?
http://michael.orlitzky.com/articles/gentoo_glep81_user_package_guidelines.xhtml#the-home-directory

Reply:
Didn't saw that article when checking for GLEP 81. In fact it isn't really needed, so I've removed it.